### ISSUE TITLE
Example codegen script fails with folders having whitespace in their names

### ISSUE THREAD

**Issue by kluufger** [apollographql/apollo-ios#253]
👋

The `API.swift` generating script from https://www.apollographql.com/docs/ios/installation.html#adding-build-step fails on folders having whitespace in their name like `User Search`:

```js
exec apollo-codegen generate ./Controllers/Users Search/UserSearch.graphql --schema 'Supporting Files/schema.json' --output API.swift
Error: ENOENT: no such file or directory, open '<full path to my app>/Controllers/Users'
  ... etc
```
I think this happens because of `$(find . -name '*.graphql')`. It doesn't put quotes around the files it finds.

Should there be a note about it for those not familiar with unix tools?

**Comment by kluufger** [comment]
Some thing like
```sh
sh-3.2$ echo $(find . -name '*.graphql' | sed -e 's/$/"/g' -e 's/^/"/g' | tr '\n' ' ')
"./Controllers/Users Search/UserSearch.graphql"
```
could have worked. But `js` seems to be trying to prepend the full path and returns
```sh
Error: ENOENT: no such file or directory, open '/Users/kluuf/Developer/testapp/"./Controllers/Users'
    at Object.fs.openSync (fs.js:660:18)
    at Object.fs.readFileSync (fs.js:565:33)
    at inputPaths.map.inputPath (/usr/local/lib/node_modules/apollo-codegen/src/loading.ts:69:21)
    at Array.map (<anonymous>)
    at Object.loadAndMergeQueryDocuments (/usr/local/lib/node_modules/apollo-codegen/src/loading.ts:68:30)
    at Object.generate (/usr/local/lib/node_modules/apollo-codegen/src/generate.ts:31:20)
    at Object.yargs.command.command.command.argv [as handler] (/usr/local/lib/node_modules/apollo-codegen/src/cli.js:191:7)
    at Object.self.runCommand (/usr/local/lib/node_modules/apollo-codegen/node_modules/yargs/lib/command.js:233:22)
    at Object.Yargs.self._parseArgs (/usr/local/lib/node_modules/apollo-codegen/node_modules/yargs/yargs.js:990:30)
    at Object.get [as argv] (/usr/local/lib/node_modules/apollo-codegen/node_modules/yargs/yargs.js:927:19)
```

**Comment by kluufger** [comment]
```
$(find `pwd` -name '*.graphql' | sed -e 's/$/"/g' -e 's/^/"/g' | tr '\n' ' ')
```
doesn't work either
```sh
sh-3.2$ apollo-codegen generate $(find `pwd` -name '*.graphql' | sed -e 's/$/"/g' -e 's/^/"/g' | tr '\n' ' ') --schema 'Supporting Files/schema.json' --output API.swift
Error: ENOENT: no such file or directory, open '/Users/kluuf/Developer/testapp/"/Users/kluuf/Developer/testapp/Controllers/Users'
```

**Comment by kluufger** [comment]
It ignores escaped whitespace as well

```sh
sh-3.2$ apollo-codegen generate $(find '.' -name '*.graphql' | sed 's/ /\\ /g') --schema 'Supporting Files/schema.json' --output API.swift
Error: ENOENT: no such file or directory, open '/Users/kluuf/Developer/testapp/Controllers/Users\'
```

**Comment by martijnwalraven** [comment]
I think you can avoid this by relying on `apollo-codegen` to do the expansion, something like `apollo-codegen generate "**/*.graphql"` should work.

**Comment by designatednerd** [comment]
This should be addressed with the changes to the build script in 0.11.0 (caveat: Watch out for the stray whitespace I fixed in #618). I'm going to close this, please open a new issue if you're still having problems. Thank you!
